packages/web/src/components/edit/fields/RemixSettingsField/TrackInfo.tsx

```diff
@@ -19,28 +17,25 @@ type TrackInfoProps = {
 
 export const TrackInfo = (props: TrackInfoProps) => {
   const { trackId } = props
-  const image = useTrackCoverArt({
-    trackId,
-    size: SquareSizes.SIZE_150_BY_150
-  })
 
-  const { data: track } = useGetTrackById({ id: trackId })
+  const { data: track } = useTrack(trackId, {
+    select: (track) => pick(track, ['title', 'owner_id'])
+  })
 
   if (!track) return null
 
-  const { user } = track
-
   return (
     <SelectedValue>
-      <DynamicImage wrapperClassName={styles.artwork} image={image} />
+      <TrackArtwork
+        trackId={trackId}
+        size={SquareSizes.SIZE_150_BY_150}
+        mr='xs'
+        css={{ minHeight: 24, minWidth: 24 }}
+      />
       <Text variant='body' strength='strong'>
-        <Text tag='span'>{track.title}</Text>{' '}
-        <Text tag='span' color='subdued'>
-          {messages.by}
-        </Text>{' '}
-        <Text tag='span'>{user.name}</Text>
+        {track.title} <Text color='subdued'>{messages.by}</Text>{' '}
```

Comment:
nit: why not put the space in the message

Reply:
good q, i think either works, i find this more readable